### PR TITLE
Add support for Laravel 9

### PR DESCRIPTION
**Pull request by latwelve**
Illuminate\Support version to support Laravel 9

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,9 +18,13 @@ jobs:
             testbench: ^5.0
           - laravel: ^8.0
             testbench: ^6.0
+          - laravel: ^9.0
+            testbench: ^7.0            
         exclude:
           - php: 7.2
             laravel: ^8.0
+          - php: 7.4
+            laravel: ^9.0
```
Comment (mvdnbrk) [inline .github/workflows/tests.yml:27]: PHP 7.3 should also be excluded with Laravel 9.

Comment (latwelve) [reply .github/workflows/tests.yml:29]: Sorry I've added that now

**Comment by latwelve** [comment]
Hey - just checking I've done everything you need from me? No rush just didn't want you waiting on something that I'd forgotten :)

**Comment by mvdnbrk** [comment]
Thanks!

**Pull request merged**